MC/config/QC/json/trd-digits-task.json

```diff
@@ -9,70 +9,139 @@
         "name": "not_applicable"
       },
       "Activity": {
-        "number": "42",
+        "number": "505618",
         "type": "2",
-        "provenance": "qc_mc",
-        "passName": "passMC",
-        "periodName": "SimChallenge"
+        "periodName": "",           "": "Period name - e.g. LHC22c, LHC22c1b_test",
+        "passName": "",             "": "Pass type - e.g. spass, cpass1",
+        "provenance": "qc",         "": "Provenance - qc or qc_mc depending whether it is normal data or monte carlo data"
       },
       "monitoring": {
         "url": "infologger:///debug?qc"
       },
       "consul": {
-        "url": "http://localhost:8500"
+        "url": "http://consul-test.cern.ch:8500"
       },
       "conditionDB": {
         "url": "ccdb-test.cern.ch:8080"
+      },
+      "infologger": { "": "Configuration of the Infologger (optional).",
+        "filterDiscardDebug": "false",    "": "Set to 1 to discard debug and trace messages (default: false)",
+        "filterDiscardLevel": "2", "": "Message at this level or above are discarded (default: 21 - Trace)" 
       }
     },
     "tasks": {
-      "QcTask": {
+      "RawDataTask": {
+        "active": "false",
+        "className": "o2::quality_control_modules::trd::RawData",
+        "moduleName": "QcTRD",
+        "detectorName": "TRD",
+        "cycleDurationSeconds": "60",
+        "maxNumberCycles": "-1",
+        "dataSource_comment": "no comment",
+        "dataSource": {
+          "type": "direct",
+          "query": "digits:TRD/DIGITS;tracklets:TRD/TRACKLETS;triggers:TRD/TRKTRGRD;rawstats:TRD/RAWSTATS"
+        },
+        "taskParameters": {
+          "peakregionstart": "7.0",
+          "peakregionend": "20.0",
+          "pulseheightpeaklower": "1.0",
+          "pulseheightpeakupper": "5.0"
+        },
+        "location": "remote",
```

Comment:
this is not applicable in the case of batch processing, you can delete all "location" lines.